### DISCUSSION TITLE
missing xkcd-external-scaler and xkcd-interceptor on Kind or Minikube

### DISCUSSION THREAD

**Discussion by iompo** [kedacore/http-add-on#335] [Q&A] [answered]
Hi,
I did run the xkcd example (following the _walkthrough.md_) on Minikube and Kind (macos). I did not get any error but I don't see the services _xkcd-external-scaler_ and _xkcd-interceptor_.
I can access the xkcd service directly,  but not through the _keda-add-ons-http-interceptor-proxy_ (so the scaling doesn't actually work). I can't figure out what the problem is, and I was wondering if somebody has some ideas on what I might have done wrong.

thanks, Marco.

**Comment by arschles** [comment, marked as answer]
Thanks @iompo for bringing this up. Are you using the `v0.2.0` release? If so, there would be no `xkcd-{external-scaler, interceptor}` services. The interceptor and external scaler in `v0.2.0` is multi-tenant and shared across all `HTTPScaledObject`s. Let me know!

**Reply by iompo** [reply]
Hi @arschles yes I'm using 0.2.0. After installing the _xkcd_ chart with elm I can see the _xkcd_ service and connect to it, but if i try to do a port forward on the _keda-add-ons-http-interceptor-proxy_ service I get a _localhost:8080 not found_. I did try to look into the different pod logs, but I did not understand what's wrong... Do you have any suggestion?

**Reply by arschles** [reply]
@iompo good to know, thanks! when you make your request to the service, you'll need to set the `Host` header to `myhost.com` (assuming you didn't change that value when you installed the helm chart). So, if you're using `curl`, the command would be:

```shell
curl -H "Host: myhost.com" localhost:8080
```

Can you let me know if this works?

**Comment by iompo** [comment]
It works, thanks @arschles. Reading the doc I did not understand how the proxy was able to forward the request to the right service, thanks again.

Marco.

**Reply by arschles** [reply]
@iompo I think that's because this is wildly unclear in the documentation, it's not your fault! do you think it would be good to add to the `walkthrough.md` document?

**Reply by iompo** [reply]
@arschles I think it would, at least for someone like me with little experience on the topic. Just by looking at the logs I couldn't figure out what the problem was, and a couple of words might have been enough to clarify it.

**Reply by arschles** [reply]
@iompo can you let me know if the documentation in https://github.com/kedacore/http-add-on/pull/347 makes sense?
